src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -181,6 +181,13 @@ private bool UsePlaceholderChannel(IRpcWorkerChannel channel)
                     return false;
                 }
 
+                // We support specialization of dotnet-isolated only on 64bit host process.
+                if (!_environment.Is64BitProcess)
+                {
+                    _logger.LogInformation("This app is configured as 32-bit and therefore does not leverage all performance optimizations. See https://aka.ms/azure-functions/dotnet/placeholders for more information.");
```

Comment:
Can you see if customers get this log message? With it being at the WebHost level, I'm suspecting they won't see it...

Try adding App Insights to an application and testing...

Reply:
Synced offline. This won't go to AI. Logging to FunctionsLogs will allow us to setup Risk alerts as suggested by @safihamid